### PR TITLE
Set up goerli testnet

### PR DESCRIPTION
**Pull request by mikemcdonald**
# Description

This adds configuration for the goerli testnet and some updated logic for features on the testnet. The two main changes are the addition of the faucet and the aave service remapping so that the testnet can pull mainnet rate info from the subgraph.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Confirm faucet works and also does not show on any other networks
- [ ] Confirm aave service data remains unchanged on mainnet

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] If package-lock.json has changes, it was intentional.
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/plugins/router.ts
Comment (garethfuller) [inline src/plugins/router.ts:148]: This is a computed property so needs .value
```suggestion
if (isGoerli.value) {
```

**Review by garethfuller — APPROVED** [review]
Looks good to me! Thanks for helping out here. 🙌 

Just a couple of minor comments which I can address if you want?

**Review by timjrobinson — APPROVED** [review]
LGTM, just some external issues with some contracts.

File: src/constants/tokens.ts
Comment (mikemcdonald) [reply src/constants/tokens.ts]: Deployed a new token and updated the `assets` repo and price mappings

**Review by timjrobinson — APPROVED** [review]

**Comment by timjrobinson** [comment]
@mikemcdonald is this ready to be merged? Alchemy is deprecating Kovan on July 1st so would be good to move over to using Goerli  before then.

**Comment by mikemcdonald** [comment]
@timjrobinson yep, let me know once merged so I can update the settings on vercel and get a beta goerli site set up as well

**Comment by timjrobinson** [comment]
Just fixed a merge conflict, no changes needed to be made. Will merge after this builds.

**Pull request merged**
